src/backend/joanie/tests/core/api/order/test_create.py

```diff
@@ -1248,7 +1370,7 @@ def test_api_order_create_authenticated_no_seats(self):
         }
         token = self.generate_token_from_user(user)
 
-        with self.assertNumQueries(47):
+        with self.assertNumQueries(70):
```

Comment:
w00t

Reply:
At this is the first request on this request, the logo thumbnail generation add extra queries.